src/main/kotlin/no/nav/pensjon/selvbetjening/fssgw/pen/PenConsumer.kt

```diff
@@ -16,15 +16,24 @@ class PenConsumer(@Value("\${pen.endpoint.url}") private val endpoint: String,
     private val log = LogFactory.getLog(javaClass)
     private val webClient: WebClient = WebClient.create()
 
-    fun callPen(body: String, callId: String?, pid: String): String {
-        val auth = auth()
+    fun callPen(urlSuffix: String, body: String, callId: String?, pid: String, fomDato: String): String {
+        return callPenClient(urlSuffix, body, callId, pid, fomDato)
+    }
 
+    fun callPen(urlSuffix: String, body: String, callId: String?, pid: String): String {
+        return callPenClient(urlSuffix, body, callId, pid, null)
+    }
+
+    fun callPenClient(urlSuffix: String, body: String, callId: String?, pid: String, fomDato: String?): String {
+        val auth = auth()
         try {
             return webClient
                     .post()
-                    .uri(endpoint)
+                    .uri(endpoint.plus(urlSuffix))
                     .header(HttpHeaders.AUTHORIZATION, auth)
                     .header("pid", pid)
+                    .header("fnr", pid)
+                    .header("fomDato", fomDato)
```

Comment:
Var litt usikker på om det er best slik eller å heller to metoder som bygger opp webclienten hver for seg på bakgrunn av ulike parametere som trengs i requestene.

Reply:
Vi kan prøve med slik du har gjort nå.